anvio/synteny.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, args, run=terminal.Run(), progress=terminal.Progress(), skip_
         self.skip_init_functions = A('skip_init_functions')
         self.genome_names_to_focus = A('genome_names')
         self.ngram_source = A("ngram_source")
-
+        self.take_first_cog = A("take_first_cog")
```

Comment:
this would be better if it was `--fist-functional-hit-only`. why focus on COG? this will work for any functional annotation source :)